src/heads-up/views/linkedin.vue

```diff
@@ -69,7 +69,7 @@
               />
               <WarningIcon
                 v-else-if="!og.image"
-                class="properties-item__icon"
+                class="properties-item__icon properties-item-icon--warning "
```

Comment:
In BEM this also needs to have the class `properties-item-icon` in order for it to be valid.